hw/hw5/Hw5_t1_oop1.py

```diff
@@ -0,0 +1,67 @@
+"""Homework 5.1."""
+import datetime
+import sys
+from typing import Any, Optional
+
+
+class Student:
+    """Create student."""
+
+    def __init__(self, first_name: str, last_name: str) -> None:
+        self.first_name = first_name
+        self.last_name = last_name
+
+    def do_homework(self, homework: "Homework") -> Optional[Any]:
```

Comment:
Can we write exact return values instead Any here and below? Any is a bad practice for typing